remediation/remediation.md

```diff
@@ -5,16 +5,18 @@ nav_order: 4
 has_children: true
 ---
 
-# Putting Nodes into a Safe State (Remediation)
+## Putting Nodes into a Safe State (Remediation)
```

Comment:
Due to markdown linter in the IDE, see _Multiple top-level headings in the same document_  ([MD025](https://github.com/DavidAnson/markdownlint/blob/v0.31.1/doc/md025.md)).

I can not make this change here (and create a top-level heading for new files of FAR) and only do that in a follow-up PR if it makes more sense.

Reply:
I don't understand. THe MD025 mentions too many top-level headings, but here there was only one

Reply:
> The MD025 mentions too many top-level headings

It also mentions:
>This rule is triggered when a top-level heading is in use (the first line of the file is an h1 heading), and more than one h1 heading is in use in the document:

Reply:
> but here there was only one

No, IIUC there is one for the title (with 1 #) coming from https://github.com/medik8s/docs/pull/63/files#diff-dfff4e442e5bc623a5e3bbd60058c511d647599ede49b9c2a876eb4b546b5d61R3

Reply:
> there is one for the title coming from #63 (files)

This is not rendered as a Title in the page though, it's the name of the file in the sidebar.

> the rest are also h1 headings (with 1 #)

I see only "Putting Nodes into a Safe State (Remediation)" as H1 heading [in the main (gh-pages) branch](https://raw.githubusercontent.com/medik8s/docs/gh-pages/remediation/remediation.md) :thinking:

Reply:
>it's the name of the file in the sidebar.

Correct and the name of the page as [title](https://github.com/medik8s/docs/blob/gh-pages/remediation/remediation.md?plain=1#L3) .
![image](https://github.com/medik8s/docs/assets/19728420/9085608c-1459-4b70-b1b8-6710483ae320)

Reply:
Yes, my point being maybe the linter is wrong 😁

Reply:
> I can not make this change here

Why not? If the linter is the problem, disable that rule 🤷🏼‍♂️ The page title is taken from the title attribute, not the heading, so that rule doesn't make sense here.

Reply:
>I can not make this change here

I basically can, even though I wrote differently. The word `not` was misplaced here 🙅🏻.
Anyway. I will not follow this remark/rule and ignore it here (and on the other new pages)